examples/getting-started-movielens/inference-HugeCTR/Triton-Inference-with-HugeCTR.ipynb

```diff
@@ -51,6 +51,48 @@
     "## Getting Started"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "37d26d48",
+   "metadata": {},
+   "source": [
+    "We need to write a configuration file with the stored model weights and model configuration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "05308361",
+   "metadata": {
+    "tags": [
+     "flake8-noqa-cell"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Overwriting /model/models/ps.json\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%writefile '/model/models/ps.json'\n",
+    "# fmt: off\n",
```

Comment:
Do we need this in there? We've removed black-nb from our pre-commit hooks because of issues like this, and I don't think that black-nb should be blocking anything (like the only reference to black-nb I see is in our requirements-dev.txt https://github.com/NVIDIA/NVTabular/search?q=black-nb and I think we can probably remove that).

Also: does this `# fmt: off` flag get written to json file? It seems to work with the hugectr json parser, but isn't strictly valid json